### DISCUSSION TITLE
Will v13 drop IE11 support?

### DISCUSSION THREAD

**Discussion by jaksenko** [vercel/next.js#34229] [Help] [answered]
IE adds big drag to final bundles:
- bigger JS because of unsupported syntax transformed by babel
- crazy HTML and JS boilerplate around `<img>` tag because of unsupported width/height aspect ratios and `loading="lazy"`

Microsoft stopped IE11 support for [Teams and 365 already](https://techcommunity.microsoft.com/t5/microsoft-365-blog/microsoft-365-apps-say-farewell-to-internet-explorer-11-and/ba-p/1591666) and will event [officially phase out support](https://docs.microsoft.com/en-us/lifecycle/faq/internet-explorer-microsoft-edge#what-is-the-lifecycle-policy-for-internet-explorer-) for the browser itself June 15th, 2022.

Is Next.js team planning to stop supporting IE11 in next major release (whenever that be)?

**Comment by dan-made** [comment, marked as answer]
According to the [RFC](https://github.com/vercel/next.js/discussions/33227) yes.

**Comment by punkpeye** [comment]
Where can I track the roadmap of v13?
